lib/menu/main_menu.cpp

```diff
@@ -39,7 +39,7 @@ MainMenu::MainMenu(int screen_width, int screen_height, GameController* game_con
     start_game_button.text.y_scale = 100;
     const auto game_controller_pass = game_controller;
     start_game_button.on_click = [game_controller_pass]() -> void {
-        game_controller_pass->startGame();
+        game_controller_pass->startGame(10);
```

Comment:
Not something which should be done in this pull request, but would we like to (or should we) make this configurable while in the lobby or will this stay 10?

Reply:
This should be configurable within the lobby however, the main menu has no ability to reach the Game system and it not able to change the max levels which why I made it a default of 10. It is not a requirement to have different values for levels.

Reply:
So I think we skip the lobby level functionality and just make it 10.

Reply:
Completely agree, we should not over engineer these things when not necessary.